frontend_app_workspace/i18next.scanner/fr/translation.json

```
@@ -1,8 +1,17 @@
 {
-  "Validate and create": "Valider et créer",
+  "Back": "Retour",
+  "Create": "Créer",
+  "Default role": "Rôle par défaut",
+  "Error while getting user spaces": "Erreur lors de l'obtention d'espaces utilisateurs",
   "Space's name": "Nom de l'espace",
   "New space": "Nouvel espace",
   "A space with that name already exists": "Un espace avec ce nom existe déjà",
   "Error while saving new space": "Erreur lors de la sauvegarde de l'espace",
+  "Next": "Suivant",
+  "None": "Aucun",
+  "Parent space:": "Espace parent :",
+  "Some input is invalid": "Certaines entrées ne sont pas valides",
```

Should we put the English text in plural to match the French and Portuguese ones?
```suggestion
  "Some inputs are invalid": "Certaines entrées ne sont pas valides",
```

---

It's a good idea! Maybe I can change to "One or more inputs are invalid" (and its translations).
What do you think @raphj?

---

I'd keep "Some inputs are invalid" as it's shorter, but I'm okay with "One or more inputs are invalid" :-)

---

Updated.